ui/dashboards/src/components/Panel/PanelActions.tsx

```diff
@@ -143,59 +180,73 @@ export const PanelActions: React.FC<PanelActionsProps> = ({
     return undefined;
   }, [editHandlers, title]);
 
-  return (
-    <HeaderActionWrapper
-      direction="row"
-      spacing={0.25}
-      alignItems="center"
-      sx={{ display: editHandlers !== undefined || readHandlers?.isPanelViewed ? 'flex' : 'var(--panel-hover, none)' }}
```

Comment:
why was this removed?

Reply:
Was there a discussion about removing the hover?

Reply:
looking at this refactoring, this is either a miss or it's not needed anymore. @andreasgerstmayr did you see that ?

Reply:
Draft PR: https://github.com/perses/perses/pull/2688